src/interpreter/bytecode-array-builder.cc

```diff
@@ -792,7 +792,6 @@ BytecodeArrayBuilder& BytecodeArrayBuilder::LoadContextSlot(
     DCHECK_EQ(mutability, kMutableSlot);
     OutputLdaContextSlot(context, slot_index, depth);
   }
-  RecordReplayAssertValue("LoadContextSlot");
```

Comment:
This and the other LoadContextSlot value seem to show up differently when replaying, probably because of how we deoptimize closure variables when replaying so they can be inspected in the debugger.